seed.ts

```diff
@@ -25,7 +25,7 @@ async function main() {
   });
 
   const musselGym = await prisma.venue.upsert({
```

Comment:
Upsert only works when the Venue already exists. Please user prisma.venue.create instead. Or just use the seed file from main branch.